src/sensor_service.cpp

```diff
@@ -190,7 +195,7 @@ tBleStatus SensorServiceClass::begin(const char *name, uint8_t addr[BDADDR_SIZE]
   }
 
   ret = aci_gatt_update_char_value(service_handle, dev_name_char_handle, 0,
-                                   strlen(name), (uint8_t *)name);
+                                   (strlen(name)<7) ? strlen(name) : 7, (uint8_t *)name);

```

Comment:
If the NULL is handle like suggested in my previous comment,
then here use the `dev_name`
```suggestion
                                   nameLen, (uint8_t *)&dev_name[1]);
```

Reply:
I integrated the change you suggested